mobile/lib/pages/common/splash_screen.page.dart

```diff
@@ -19,45 +18,22 @@ class SplashScreenPage extends HookConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final apiService = ref.watch(apiServiceProvider);
     final serverUrl = Store.tryGet(StoreKey.serverUrl);
+    final endpoint = Store.tryGet(StoreKey.serverEndpoint);
     final accessToken = Store.tryGet(StoreKey.accessToken);
     final log = Logger("SplashScreenPage");
 
     void performLoggingIn() async {
```

Comment:
Is there any way to put this function somewhere else? I typically don't expect to find lots of business logic in the build function of a widget. Maybe we can add this function to a relevant provider. The ui should really just be watching the state of the notifier and updating accordingly.

Reply:
I appreciate that, but also this is the current state, we are trying to fix the problem without creating more problems by refactoring stuff too much.